boa/src/exec/mod.rs

```diff
@@ -357,6 +357,20 @@ impl Executor for Interpreter {
                 Ok(match *op {
                     UnaryOp::Minus => to_value(-v_a.to_num()),
                     UnaryOp::Plus => to_value(v_a.to_num()),
+                    UnaryOp::IncrementPost => {
+                        self.set_value(a.deref(), to_value(v_a.to_num() + 1.0))?;
```

Comment:
Can we use `*a` instead of `a.deref()` here?
Also, can we pass the `op` without the `ref` in the match? I think `UnaryOp` implements `Copy`, and if not, it probably should :)

Reply:
`a` here is `&Box<Node>`, so instead of `a.deref()` it will be `&*a`. `deref` looks cleaner to me, what do you think?

Reply:
Ok, perfect to me :)